src/vs/workbench/browser/parts/views/customView.ts

```diff
@@ -530,136 +596,133 @@ export class CustomTreeView extends Disposable implements ITreeView {
+			return this.withProgress(this.treeView.dataProvider.getChildren(element));
 		}
 		return Promise.resolve([]);
 	}
-
-	shouldAutoexpand(tree: ITree, node: ITreeItem): boolean {
-		return node.collapsibleState === TreeItemCollapsibleState.Expanded;
-	}
-
-	getParent(tree: ITree, node: any): Promise<any> {
-		return Promise.resolve(null);
-	}
-}
-
-interface ITreeExplorerTemplateData {
-	resourceLabel: IResourceLabel;
-	icon: HTMLElement;
-	actionBar: ActionBar;
-	aligner: Aligner;
 }
 
 // todo@joh,sandy make this proper and contributable from extensions
 registerThemingParticipant((theme, collector) => {
 
 	const findMatchHighlightColor = theme.getColor(editorFindMatchHighlight);
 	if (findMatchHighlightColor) {
-		collector.addRule(`.file-icon-themable-tree .monaco-tree-row .content .monaco-highlighted-label .highlight { color: unset !important; background-color: ${findMatchHighlightColor}; }`);
+		collector.addRule(`.file-icon-themable-tree .monaco-list-row .content .monaco-highlighted-label .highlight { color: unset !important; background-color: ${findMatchHighlightColor}; }`);
 		collector.addRule(`.monaco-tl-contents .monaco-highlighted-label .highlight { color: unset !important; background-color: ${findMatchHighlightColor}; }`);
 	}
 	const findMatchHighlightColorBorder = theme.getColor(editorFindMatchHighlightBorder);
 	if (findMatchHighlightColorBorder) {
-		collector.addRule(`.file-icon-themable-tree .monaco-tree-row .content .monaco-highlighted-label .highlight { color: unset !important; border: 1px dotted ${findMatchHighlightColorBorder}; box-sizing: border-box; }`);
+		collector.addRule(`.file-icon-themable-tree .monaco-list-row .content .monaco-highlighted-label .highlight { color: unset !important; border: 1px dotted ${findMatchHighlightColorBorder}; box-sizing: border-box; }`);
 		collector.addRule(`.monaco-tl-contents .monaco-highlighted-label .highlight { color: unset !important; border: 1px dotted ${findMatchHighlightColorBorder}; box-sizing: border-box; }`);
 	}
 	const link = theme.getColor(textLinkForeground);
 	if (link) {
 		collector.addRule(`.tree-explorer-viewlet-tree-view > .message a { color: ${link}; }`);
 	}
-	const focustBorderColor = theme.getColor(focusBorder);
-	if (focustBorderColor) {
-		collector.addRule(`.tree-explorer-viewlet-tree-view > .message a:focus { outline: 1px solid ${focustBorderColor}; outline-offset: -1px; }`);
+	const focusBorderColor = theme.getColor(focusBorder);
+	if (focusBorderColor) {
+		collector.addRule(`.tree-explorer-viewlet-tree-view > .message a:focus { outline: 1px solid ${focusBorderColor}; outline-offset: -1px; }`);
 	}
 	const codeBackground = theme.getColor(textCodeBlockBackground);
 	if (codeBackground) {
 		collector.addRule(`.tree-explorer-viewlet-tree-view > .message code { background-color: ${codeBackground}; }`);
 	}
 });
 
-class TreeRenderer implements IRenderer {
+interface ITreeExplorerTemplateData {
+	elementDisposable: IDisposable;
```

Comment:
what is this `elementDisposable`?

Reply:
Copied from other trees, but it is not needed here. removed.